xclim/sdba/adjustment.py

```diff
@@ -1204,6 +1213,229 @@ def _adjust(
         return out
 
 
+class OTC(Adjust):
+    r"""Optimal Transport Correction.
+
+    Following :cite:t:`sdba-robin_2019`, this multivariate bias correction method finds the optimal transport
+    mapping between simulated and observed data. The correction of every simulated data point is the observed
+    point it is mapped to.
+
+    See notes for an explanation of the algorithm.
+
+    Parameters
+    ----------
+    bin_width : list | None
+        Bin widths for all dimensions.
+        Will be estimated by default.
+    bin_origin : list | None
+        Bin origins for all dimensions.
+        Default is 0.
+    num_iter_max : int | None
+        Maximum number of iterations used in the earth mover distance algorithm.
+        Default is 100_000_000.
+    spray_bins : bool
+        If `False`, output points are located at the center of their bin.
+        If `True`, a random location is picked uniformly inside their bin. Default is `True`.
```

Comment:
@SarahG-579462 @coxipi 
Not sure about `spray_bins`. Contenders include `jitter_inside_bin` and `randomize_bin`.
Apart from that, I think it's ready.

Reply:
I like jitter_inside_bins, personally